src/transports/hello_http_client.erl

```diff
@@ -95,13 +95,13 @@ content_type(Signarute) ->
 http_send(Client, Request, Signarute, State = #http_state{url = URL, options = Options}) ->
     #http_options{method = Method, ib_opts = Opts} = Options,
     {ok, Vsn} = application:get_key(hello, vsn),
-    Headers = [{<<"Content-Type">>, content_type(Signarute)},
-               {<<"Accept">>, content_type(Signarute)},
-               {<<"User-Agent">>, <<"hello/", (list_to_binary(Vsn))/binary>>}],
+    Headers = [{<<"content-type">>, content_type(Signarute)},
+               {<<"accept">>, content_type(Signarute)},
+               {<<"user-agent">>, <<"hello/", (list_to_binary(Vsn))/binary>>}],
```

Comment:
i would prefer to split it into two commits and PRs. a PR is the semantic grouping of commits implementing a change.
a PR is not a grouping of unrelated commits.